crates/context_aware_config/src/api/default_config/handlers.rs

```diff
@@ -1,35 +1,45 @@
 extern crate base64;
 use super::types::CreateReq;
 use service_utils::helpers::validation_err_to_str;
-use service_utils::{bad_argument, unexpected_error, validation_error};
+use service_utils::{
+    bad_argument, db_error, not_found, unexpected_error, validation_error,
+};
 
 use superposition_types::{SuperpositionUser, User};
 
 use crate::api::context::helpers::validate_value_with_function;
 use crate::{
     api::functions::helpers::get_published_function_code,
-    db::{self, models::DefaultConfig, schema::default_configs::dsl::default_configs},
+    db::{
+        self,
+        models::{Context, DefaultConfig},
+        schema::{contexts::dsl::contexts, default_configs::dsl::default_configs},
+    },
     helpers::validate_jsonschema,
 };
 use actix_web::{
-    get, put,
-    web::{self, Data, Json},
+    delete, get, put,
+    web::{self, Data, Json, Path},
     HttpResponse, Scope,
 };
 use chrono::Utc;
 use diesel::{
+    delete,
     r2d2::{ConnectionManager, PooledConnection},
     ExpressionMethods, PgConnection, QueryDsl, RunQueryDsl,
 };
 use jsonschema::{Draft, JSONSchema, ValidationError};
-use serde_json::{json, Value};
+use serde_json::{from_value, json, Map, Value};
 use service_utils::{
     result as superposition,
     service::types::{AppState, DbConnection},
 };
 
 pub fn endpoints() -> Scope {
-    Scope::new("").service(create).service(get)
+    Scope::new("")
+        .service(create)
+        .service(get)
+        .service(delete_default_config)
```

Comment:
```suggestion
        .service(delete)
```

Reply:
delete name collides with the diesel::delete function , thats why kept this as delete_default_config,
we might use diesel::delete at multiple places , so its better to rename one time used declarations

Reply:
+1, @pratikmishra356 can you make the changes

Reply:
I guess this is resolved

Reply:
its already done